install

```diff
@@ -145,6 +145,11 @@ def chgrp?(path)
   !File.grpowned?(path)
 end
 
+# The USER env var is not guaranteed to be available, so fall back to id if not
+# provided. We destructively set it in the environment to make it available to
```

Comment:
I feel that `destructively` is a bit misleading, because it's only set if it's not already set. I'd be inclined to remove this word.

Reply:
Yep, I agree. I went to change this and saw that @MikeMcQuaid had fixed it - thanks 👍.